packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -15,22 +18,16 @@ import {
   getSerifMedium,
 } from '@bbc/psammead-styles/font-styles';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import {
+  getPica,
+  getGreatPrimer,
+  getLongPrimer,
```

Comment:
👍 Updates to typography settings look good! 
The summary typography matches that of StoryPromo's Summary. 
The TV bulletin headline typography matches the StoryPromo Leading heading.
The Radio bulletin headline typography matches the StoryPromo regular heading.